### DISCUSSION TITLE
sf config get --value parameter

### DISCUSSION THREAD

**Discussion by pkozuchowski** [forcedotcom/cli#3044] [Ideas]
This is proposal for new `--value` parameter for `sf config get` command, that would return only value of the configuration variable.

Example use case, assuming target-org is `myScratchOrg`
```bash
TARGET_ORG="$(sf config get --value target-org )"

$ echo $TARGET_ORG
myScratchOrg
```

**Comment by mdonnalley** [comment]
I'll let @VivekMChawla speak to whether or not we should prioritize implementing this but it's worth pointing out that you can achieve this today using `jq`

```
sf config get target-org --json | jq '.result[0].value'
```

**Reply by pkozuchowski** [reply]
Yup, that was one of the directions I was investigating. My main use case is for scripts which my team use to set up scratch orgs and seed it with data. We are working on rather limited remote desktops, so JQ solution would require everyone to download jq and put it in PATH, since it's not standard in windows git bash.

**Reply by shetzel** [reply]
`jq` is included in our full docker image if that is an option for you

**Reply by daveespo** [reply]
I'll admit that we have similar environment constraints to @pkozuchowski  and adding a dependency on `jq` is a drag. 

Returning the isolated config value would simplify (eliminate) a terrible regex we use to fish out the config value from the JSON response.

**Reply by cristiand391** [reply]
I would love to have a `jq` query parser for javascript, we could have a `--jq <jq-query>` in our commands which could help in these restricted-machines scenarios, like the GH CLI commands do:
https://cli.github.com/manual/gh_codespace_list

example:
```
sf config get target-org --json --jq ".result[0].name"
test-blablabla
```

**Reply by daveespo** [reply]
Yep, that would fit the bill. Approved! :-)

**Reply by cristiand391** [reply]
just saw this new project yesterday, I think it opens the door for a `--jq` flag 😉 
https://github.com/platformatic/fgh
